src/sentry/models/groupsubscription.py

```diff
@@ -123,7 +144,7 @@ def get_participants(self, group) -> Mapping[ExternalProviders, Mapping[Any, int
         return result
 
 
-class GroupSubscription(Model):
+class GroupSubscription(Model):  # type: ignore
```

Comment:
mypy only knows about classes if they're in the same "run". The file with `Model` is not in `mypy.ini` so it infers it is `Any`. If on the other hand the class _is_ included and there is a `#type: ignore` it will return a warning "unused 'type: ignore'" so I remove it in other places.